src/ring_theory/valuation/basic.lean

```diff
@@ -424,14 +424,14 @@ begin
       by_contra h_1,
       cases ne_iff_lt_or_gt.1 h_1,
       { simpa [hh, lt_self_iff_false] using h.2 h_2 },
-      { rw [← inv_one, eq_inv_iff_eq_inv, ← map_inv₀] at hh,
-        exact hh.le.not_lt (h.2 ((one_lt_val_iff v' hx).1 h_2)) } },
+      { rw [← inv_one, ← inv_eq_iff_eq_inv, ← map_inv₀] at hh,
+        exact hh.symm.le.not_lt (h.2 ((one_lt_val_iff v' hx).1 h_2)) } },
```

Comment:
```suggestion
        exact hh.not_gt (h.2 ((one_lt_val_iff v' hx).1 h_2)) } },
```

Reply:
OK, thanks.

Reply:
That actually does not work. Maybe `.not_lt`; I'll check...